config-model-generator/Makefile

```diff
@@ -7,8 +7,6 @@ docker_tag:
 all: docker_build docker_push
 
 clean: java_clean
-	rm ../cluster-operator/src/main/resources/kafka-*-config-model.json || true
-	./build-config-models.sh clean
```

Comment:
I can understand why you remove the deletion of the model files as you moved it in the script. But nothing is now calling the script, or?